HelpSource/Guides/News-3_13.schelp

```diff
@@ -1,8 +1,8 @@
-title:: News in 3.13
-summary:: A summary of news in SC 3.13
+title:: News in 3.13.1
+summary:: A summary of news in SC 3.13.1
 categories:: News
 
-Welcome to the SuperCollider 3.13 release! 
+Welcome to the SuperCollider 3.13.1 release! 
 
```

Comment:
I think in the past we didn't include patch release numbers here, I think this could stay as 3.13.

The addition below ("3.13.1 fixes an allocation bug...") should of course stay.